pkgs/desktops/cinnamon/cinnamon-gsettings-overrides/default.nix

```diff
@@ -35,9 +36,18 @@ let
     cinnamon-session
     cinnamon-settings-daemon
     cinnamon-common
+    gnome.gnome-terminal
     gtk3
   ] ++ extraGSettingsOverridePackages;
 
+  gsettingsOverrides = ''
+    # https://github.com/NixOS/nixpkgs/issues/200017
```

Comment:
you could also directly mention the fedora defaults aswell, but since they're linked in the issue aswell idk if it's necesarry